app/views/support_interface/performance/index.html.erb

```diff
@@ -5,3 +5,14 @@
 <p class='govuk-body'>
   <%= govuk_link_to 'Performance dashboard', integrations_performance_path %> (public)
 </p>
+
+<h2 class='govuk-heading-l'>Downloads for analysis</h2>
+
+<h3 class='govuk-heading-m'>Application timings</h3>
+
+<p class='govuk-body'>
+  The application timings provides data on when a application
+  form attribute was last updated by the candidate.
+</p>
+
+<%= govuk_link_to 'Download application timings (CSV)', support_interface_application_timings_path %>
```

Comment:
This looks oddly small on the page — should it be inside `govuk-body`?

<img width="990" alt="Screenshot 2020-02-18 at 15 24 43" src="https://user-images.githubusercontent.com/642279/74749946-d7dbcb80-5262-11ea-8588-ad4e987b6723.png">

Reply:
Fixed!